src/server.cpp

```diff
@@ -967,14 +986,87 @@ namespace quicr {
                     }
                 }
 
-                PublishReceived(
-                  conn_ctx.connection_handle,
-                  msg.request_id,
-                  tfn,
-                  { { 0, msg.group_order, std::chrono::milliseconds(0), 0, new_group_request_id }, msg.track_alias });
+                PublishReceived(conn_ctx.connection_handle,
+                                msg.request_id,
+                                tfn,
+                                { { 0,
```

Comment:
what is this 0 for

Reply:
For posterity, this is here because publish attributes are a superset of subscribe attributes, which is not actually ideal for things like subscriber priority here. Will open an issue to resolve that relationship.